huaweicloud/services/acceptance/cce/resource_huaweicloud_cce_partition_test.go

```diff
@@ -0,0 +1,139 @@
+package cce
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	"github.com/chnsz/golangsdk/openstack/cce/v3/partitions"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+)
+
+func getPartitionResourceFunc(conf *config.Config, state *terraform.ResourceState) (interface{}, error) {
+	c, err := conf.CceV1Client(acceptance.HW_REGION_NAME)
+	if err != nil {
+		return nil, fmt.Errorf("error creating CCE v1 client: %s", err)
+	}
+	resp, err := partitions.Get(c, state.Primary.Attributes["cluster_id"],
+		state.Primary.Attributes["availability_zone"]).Extract()
```

Comment:
```suggestion
		state.Primary.ID).Extract()
```
it's clearer to use ID than availability_zone although they have the same value.

Reply:
That make sense, I've modified this code.